src/platform/Zephyr/CHIPDevicePlatformConfig.h

```diff
@@ -72,10 +104,12 @@
 #endif // CHIP_DEVICE_CONFIG_SERVER_SHUTDOWN_ACTIONS_SLEEP_MS
 
 #ifndef CHIP_DEVICE_CONFIG_HEAP_STATISTICS_MALLINFO
-#if !defined(CONFIG_CHIP_MALLOC_SYS_HEAP) && defined(CONFIG_NEWLIB_LIBC)
+#if !defined(CONFIG_CHIP_MALLOC_SYS_HEAP) && (defined(CONFIG_NEWLIB_LIBC) || (defined(CONFIG_NEWLIB_LIBC_NANO)))
```

Comment:
`NEWLIB_LIBC_NANO` depends on `NEWLIB_LIBC` so this change is not needed, right?

Reply:
I guess, too, just need some feedbacks. It's like one of my first PR as an intern, and I probably will do some errors

Reply:
That's OK, I was just curious if you had any build issues without it.